pkg/mockfilesystem/main.go

```diff
@@ -63,8 +64,54 @@ func SetupBundleTemplate(rootTemplateDir string, fs afero.Fs) error {
 	return nil
 }
 
+func SetupBicepBundle(rootDir string, fs afero.Fs) error {
```

Comment:
Are we recreating the existing files in a virtual file system? Is there a reason to do it this way? It seems it would be easier to just setup the test dir to be the proper structure and read from there.